Monika After Story/game/event-handler.rpy

```diff
@@ -1459,6 +1461,112 @@ init -1 python in evhand:
         )
     )
 
+
+    class EventListItem(object):
+        """
+        Representation of an EventListItem (ELI)
+        """
+        # default values for each main param
+        # make sure this matches push/queue
+        DEFAULT_VALUES = (
+            False, # notify
+            None, # context
+        )
+
+        ITEM_LEN = len(DEFAULT_VALUES) + 1 # defaults + event label
+
+        IDX_EVENT_LABEL = 0
+        IDX_NOTIFY = 1
+        IDX_CONTEXT = 2
+
+        def __init__(self, data):
+            """
+            Constructor
+
+            IN:
+                data - the data directly from event list
+            """
+            self._eli = data
+
+        @staticmethod
+        def build(evl, *args):
+            """
+            Builds an ELI.
+
+            IN:
+                evl - event label
+                *args - the other args for an EventListItem.
+
+            RETURNS: EventListItem object
+            """
+            return EventListItem(EventListItem._build_raw(evl, *args))
```

Comment:
my threshold for using classmethod is when I get really fed up typing the class name. Twice (or 3x) is not enough to reach that threshold, so I'm fine leaving it static.

The constructor is for direct object wrapper from `event_list`. The static build methods are for when passing in each data element (or some of the data elements).

`_build_raw` just generates the raw data tuple, originally separate because I was gonna use it for just push/queue, but then changed those functions to use `build` instead so we can leverage a single code path for raw data format (`_raw`). 

I'd rather keep them separate just in case.